### PR TITLE
Create and update PRS records for BLorient9798

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Owners and qene authors (or collectors)

### CODE REVIEW

File: PRS9001-10000/PRS9430Tewodros.xml [hunk 1]
```diff
@@ -45,15 +46,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <body>
          <listPerson>
             <person sex="1" sameAs="wd:Q262589">
-               <persName>
-                  
-                  Tewodros II
-               </persName>
-               <persName type="alt">Kaśa of Qwara</persName>
-               <persName type="alt">Kaśa Ḫaylu</persName>
+               <persName xml:id="n1" xml:lang="gez" type="regnal">ቴዎድሮስ፡ </persName>
```
Comment (DenisNosnitsin1970) [inline PRS9001-10000/PRS9430Tewodros.xml:49]: please check space before the closing bracket </

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/PRS14499GabraMasih.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        <body>
+            <listPerson>
+                <person>
+                    <persName xml:lang="gez" xml:id="n1">ገብረ፡ መሲሕ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Gabra Masiḥ</persName>
+                    <occupation type="literary">Author of qǝne poems.</occupation>
+                    <floruit notBefore="1700" notAfter="1868">18th/19th century</floruit>
+                </person>
+                <listRelation>
+                    <relation name="saws:isAttributedToAuthor" active="PRS14499GabraMasih" passive="LIT0000000000000000000"/>
```
Comment (DenisNosnitsin1970) [inline new/PRS14499GabraMasih.xml:54]: isAttributedToAuthor is for literary works. I use relation name="dcterms:creator". Please first make LIT record and include it.

File: new/PRS14500Ewostatewos.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ኤዎስጣቴዎስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾEwosṭātewos</persName>
+                    <occupation type="literary">Author of qǝne poems.</occupation>
+                    <floruit notBefore="1700" notAfter="1868">18th/19th century</floruit>
+                </person>
```
Comment (DenisNosnitsin1970) [inline new/PRS14500Ewostatewos.xml:52]: Why Ewostatewos is the author of the qene poem quoted by Strelcyn? (Sorry, you asked about this, I probably misunderstood your question). ʾEwosṭātewos is not indicated here to be the author, rather, the poem is about him (and then it cannot be excluded that it is about the monastic leader from the north, who initiated Ewostatean movement). Authorship indication in qene, if available, is quite clear.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/PRS14500Ewostatewos.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14500Ewostatewos.xml:52]: I supposed, that the mentioned ʾEwosṭātewos was the one, who alledgedly composed or collected them. If this is unlikely, I think, I should delete this record (PRS14500Ewostatewos) and explain the possible reference to PRS3934Ewostate in a note in the manuscript record.

Comment (karljonaskarlsson) [reply new/PRS14500Ewostatewos.xml:52]: I would also think that the mentioned ʾEwosṭātewos is more likely the topic of the qene.

**Review by karljonaskarlsson — APPROVED** [review]

**Review by karljonaskarlsson — APPROVED** [review]

**Pull request merged**